napari_allencell_annotator/_tests/fakes/fake_viewer.py

```diff
@@ -24,3 +25,14 @@ def alert(self, alert_msg: str) -> None:
 
     def get_layers(self) -> List[Layer]:
         return self._layers
+
+    def get_all_points_layers(self) -> List[Points]:
+        return self._points_layers
+
+    def create_points_layer(self, name: str, color: str, visible: bool) -> Points:
+        points: Points = Points(data=None, name=name, color=color, visible=visible)
+        self._points_layers.append(points)
+        return points
```

Comment:
this would be closer to the real thing if the points layer was added to the actual layer list. Which is what happens in napari

Reply:
changed in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/39abde6e5970471afef7c4250b554f73feda98be